prebuild/Debian/preinstall.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+sudo apt-get update
+sudo apt-get install -y build-essential libcairo2-dev libjpeg-dev libpango1.0-dev libgif-dev librsvg2-dev libpixman-1-dev patchelf
```

Comment:
@chearon did you build all of the Linux deps from source for a reason, or can we use precompiled packages as this PR does?

Reply:
Yes, we need to build on an old OS for maximum compatibility. Programs built against a newer libc may have symbols that don't exist on older OSes (this happened to real users). But we want newer libraries, so have to build from source.

Reply:
> But we want newer libraries

👍 forgot about this part.